modules/manage/pages/schema-reg/schema-reg-api.adoc

```diff
@@ -931,6 +931,53 @@ Curl::
 --
 ====
 
+== Use READONLY mode for disaster recovery
+
+The `/mode` endpoint allows you to put the Schema Registry in read-only or read-write mode. A read-only Schema Registry does not accept direct writes. An active production cluster replicates schemas to the read-only Schema Registry to keep it in sync, for example using MirrorMaker2. In a failover due to a disaster or outage, you can set Schema Registry to read-write mode, taking over for the failed cluster and ensuring availability. 
```

Comment:
@kbatuigas  we should say 'can replicate schemas'. THis is all DIY and not built into the product.

Reply:
"An active production cluster can replicate schemas to the read-only Schema Registry to keep it in sync, but users in the DR site will not be able to update schemas directly, ensuring the DR cluster has an exact replica of the schemas in production.  In a failover ..."